README.md

```diff
@@ -142,12 +142,10 @@ standards (no plugins, just HTML5 and WebRTC)! It's easy to get started!
 ##### Downloading a file is simple:
 
 ```js
-var WebTorrent = require('webtorrent')
-
 var client = new WebTorrent()
-var magnetURI = '...'
+var torrentId = 'magnet:?xt=urn:btih:6a9759bffd5c0af65319979fb7832189f4f3c35d&dn=sintel.mp4&tr=wss%3A%2F%2Ftracker.btorrent.xyz&tr=wss%3A%2F%2Ftracker.fastcast.nz&tr=wss%3A%2F%2Ftracker.openwebtorrent.com&tr=wss%3A%2F%2Ftracker.webtorrent.io&ws=https%3A%2F%2Fwebtorrent.io%2Ftorrents%2Fsintel-1024-surround.mp4'
```

Comment:
Please look at `client.add` API: `client.add(torrentId, [opts], [function ontorrent (torrent) {}])`

Every magnet URI is torrent id. But not every torrent id is magnet URI.

`torrentId` can be one of:

- magnet uri (string)
- torrent file (buffer)
- info hash (hex string or buffer)
- parsed torrent (from [parse-torrent](https://github.com/feross/parse-torrent))
- http/https url to a torrent file (string)
- filesystem path to a torrent file (string)

E.g. you can change `var magnetURI = 'magnet:?xt=urn:btih:6a9759bffd5c0af65319979fb7832189f4f3c35d'` to `magnetURI = 'https://webtorrent.io/torrents/sintel.torrent'` in this example and the resulting code will still work, bacause torrentId argument of client.add is not limited only to magnet URIs, but in this case using `magnetURI` as a name of variable is not a good idea, because actually it is not a magnet URI, it is URL to *torrent file.